de.haeckerfelix.Remotely.json

```
@@ -0,0 +1,84 @@
+{
+    "app-id" : "de.haeckerfelix.Remotely",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.32",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "remotely",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libgpg-error",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.27.tar.gz",
+                    "sha256" : "04bdc7fd12001c797cc689b007fe24909f55aa0ee1d6d6aef967d9eebf5b2461"
+                }
+            ]
+        },
+        {
+            "name" : "libgcrypt",
+            "config-opts" : [
+                "--disable-doc"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/gpg/libgcrypt/archive/libgcrypt-1.8.2.tar.gz",
+                    "sha256" : "ddad335a5cf73c812dd06bb0ce6af86a803333fc2f98ae86a449ea02e0dda0f9"
+                }
+            ]
+        },
+        {
+            "name" : "gtkvnc",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.gnome.org/sources/gtk-vnc/0.7/gtk-vnc-0.7.1.tar.xz",
+                    "sha256" : "f34baa696615ef67666e8465b4d0ac563355e999a77d2cc42ad4625a24f7aab1"
+                }
+            ]
+        },
+        {
+            "name" : "remotely",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--libdir=lib"
+            ],
+            "builddir" : true,
```

```suggestion
            "builddir" : true,
```
Same as the two other comments.